### PR TITLE
automated BDD test specs for Pill, Pod, Portrait, Preview

### PR DESCRIPTION
**Pull request by DlgSHi**
**automated BDD test specs for:**

- Pill, 
- Pod, 
- Portrait, 
- Preview

fix for a cucumberautocomplete.steps value

### CODE REVIEW

File: cypress/integration/pill.feature [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  Background: Open Pill component page
+    Given I open "Pill" component page
+
+  @positive
+  Scenario Outline: Change Pill children to <children>
+    When I set children to "<children>"
+    Then Pill children on preview is set to "<children>"
+    Examples:
+      | children               |
```
Comment (dawidzarzycki) [inline cypress/integration/pill.feature]: Can we fix formatting please?

File: cypress/integration/pod.feature [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    When I check border checkbox
+      And I uncheck border checkbox
+    Then Pod component has no border
+  
+  @positive
+  Scenario Outline: Change Pod children to <children>
+    When I set children to "<children>"
+    Then Pod children on preview is set to "<children>"
+    Examples:
+      | children               |
```
Comment (dawidzarzycki) [inline cypress/integration/pod.feature]: Fix formatting :)

File: cypress/locators/pod/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import {
+  POD_PREVIEW, POD_EDIT_ICON, POD_FOOTER,
+  POD_CHILDREN,
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/locators/pod/index.js]: fixed

File: cypress/integration/preview.feature [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      # @ignore because of FE-1447
+      # | <>                       |
+
+  @positive
+  Scenario Outline: Set width to <width>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/preview.feature]: fixed

File: cypress/integration/preview.feature [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    Then Preview width is set to "<width>"
+    Examples:
+      | width |
+      | 0px   |
+      | 1px   |
+      | 10px  |
+      | 100px |
+
+  @positive
+  Scenario Outline: Set height to <height>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/preview.feature]: fixed

File: cypress/integration/preview.feature [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    Then Preview height is set to "<height>"
+    Examples:
+      | height |
+      | 0px    |
+      | 1px    |
+      | 10px   |
+      | 100px  |
+
+  @positive
+  Scenario Outline: Set lines to <lines>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/preview.feature]: fixed

File: cypress/support/step-definitions/portrait-steps.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+      portraitPreview()
+        .should('have.class', `${PORTRAIT_CLASS}${property}`);
+      break;
+    default:
+      throw new Error('Not a Portrait component property');
+  }
+});
+
+
+// Portrait shape on preview is
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/portrait-steps.js]: fixed

File: cypress/integration/pod.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/integration/pod.feature]: fixed

File: cypress/integration/pill.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/integration/pill.feature]: fixed

**Pull request merged**